### ISSUE TITLE
Error in CodeListener/0.1.7.0/AutoComplete/Rhino/init.py

### ISSUE THREAD

**Issue by yck011522** [ccc159/PythonScript#5]
I had the problem where code completion is not working for Rhino.RhinoDoc. Upon investigation I noticed there is two flagged error in the init.py. Two function signatures have a line like this:
```
def TryGetEnumValue(self,*__args):
# Error generating skeleton for function TryGetEnumValue: Method must be called on a Type for which Type.IsGenericParameter is false.

def GetEnumValue(self,*__args):
# Error generating skeleton for function GetEnumValue: Method must be called on a Type for which Type.IsGenericParameter is false.
```

I use Pylance and it is flagging these two lines as error because there is nothing in the `def` signature after this error message. I added a `pass` to them and it seems to work fine.  I suspect this fix is related to the problem in #4

**Comment by yck011522** [comment]
Oh I just realize this issue might be better posted to https://github.com/ccc159/CodeListener
Anyways. Feel free to close this and move it.

**Comment by jingcheng-chen** [comment]
The auto-complete feature fully depends on the stubs, as python cannot parse CLR assemblies.

If `Rhino.RhinoDoc` is not [here](https://github.com/gtalarico/ironpython-stubs/tree/master/release/stubs/Rhino), then there's not so much you can do.

**Comment by yck011522** [comment]
@ccc159 I'm not sure what you mean. 
The stubs I refer to (that has the two lines of error) is the stubs that comes with CodeListener Installation.

**Comment by jingcheng-chen** [comment]
Yes, those stubs in my plugin come from the link I mentioned above. I didn't bother to re-make it. Of course if you are interested, you could try to make a new one from current Rhino assemblies.
